src/main/java/com/tropicalbastos/boids/settings/Platform.java

```diff
@@ -9,11 +9,7 @@ public static String getOsName() {
       return OS;
    }
    
-   public static boolean isWindows() {
```

Comment:
Also, no need to remove this, perhaps we'll need it later?

Reply:
Good point, I'll add that in tomorrow, gotta catch some z's before work tomorrow!